_releases/2023-07-18-june-2023.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+category: releases
+title: June 2023 Release Notes
+filename-custom: june-2023
+last_modified_at: 
+sidenav: about_us
+release-year: "2023"
+description: "Release notes for June 2023, including several improvements and one fix."
+---
+## Improvements
+
+* **Results Page Redesign**: We've continued progress on updating our frontend components.
+  * We built the [Graphics Best Bets](https://search.gov/admin-center/content/best-bets.html#best-bets-graphics) component for the new results page.
+  * We added term-match highlighting to the backend data for results snippets.
+  * We made sure that long URLs and descriptions display in a user-friendly manner.
+  * We prepared backend data for some types of search results:
+    * Jobs 
+    * Federal Register documents
+    * Health topics
+* **Indexing improvements**: Our system now prioritizes customer requests for reindexing over general index maintenance.
+* **Results matching improvements**: In this phase we are looking at alphanumeric strings, such as form numbers or policy numbers. We created a Levenstein model that will identify variations on strings that are "close enough" to consider matches. For example, "DD 214" and "DD214."
```

Comment:
pretty sure the spelling is Levenshtein ref: https://en.wikipedia.org/wiki/Levenshtein_distance